SIL.Core.Tests/Xml/XmlUtilsTests.cs

```diff
@@ -176,6 +176,37 @@ public void WriteNode_DoesNotIndentChildWhenSuppressed()
 			}
 			Assert.That(output.ToString(), Is.EqualTo(expectedOutput));
 		}
+
+		[Test]
+		public void WriteNode_PreserveNamespacesArePreserved()
+		{
+			string input = @"<text><span class='bold' xmlns:fw='https://fakefw.com' fw:special='no' xml:space='preserve'> </span></text>";
+			string expectedOutput =
+				"<?xml version=\"1.0\" encoding=\"utf-16\"?>\r\n"
+				+ "<root>\r\n"
+				+ "	<text>\r\n"
+				+ "		<span\r\n"
+				+ "			class=\"bold\" xmlns:fw=\"https://fakefw.com\"\r\n"
```

Comment:
so based on the test failure this line looks like this in net standard
`class=\"bold\"\r\n\t\t\txmlns:fw=\"https://fakefw.com\"\r\n`
so there's a newline between `class="bold"` and xmlns, but not on framework. I debugged this and that's just how the code is written, I don't really see a good way around this. Even reordering the namespace doesn't make a difference.
This is because there's a special case for `xmlns` specifically. 

Looking into the source indenting/formatting works with via overloads. Internally `WriteStartNamespaceDeclaration` is called which is used only for `xmlns` in dotnet standard this method is overloaded in `XmlEncodedRawTextWriterIndent` to indent the line, and in framework it's not. Sadly these are internal so we can't just provide our own version.

I suspect this will be a problem as it will thrash the xml when run on one version of dotnet vs another. I'm not sure how, but if we could use the netstandard version in framework then it would resolve this issue.